### ISSUE TITLE
Meeting reminders can't be dismissed without restarting TB

### ISSUE THREAD

**Issue by weknowsoftware** [ExchangeCalendar/exchangecalendar#217]
This is similar to issues that occurred on previous versions but then went away in v4.x

I have a recurring meeting reminder via an Exchange calendar. It pops up the reminder on time but every time I dismiss it, it pops back up. Also, it can suddenly start doing this for past (dismissed!) reminders about 12 hours after the meeting time.

In both cases, restarting Thunderbird makes them go away.

**Comment by nparsons999** [comment]
I see this problem too.
I also see a few PAST meeting pop-ups which can't be permanently dismissed.
Sometimes, I have to exit TB AND type killall thunderbird to get rid of the persistent pop-up reminders.
launch TB and it's OK again (for a while anyway). I have to restart/killall TB approx three times/day.
Ubuntu 18.4.1 LTS very latest TB/lightning/exchangecalendat releases

**Comment by zioalex** [comment]
I had the same problem but not anymore now with the latest v5.0.0-alpha2

**Comment by nparsons999** [comment]
Right. I thought I was using the latest alpha2 but I'll do a fresh clone/checkout/build etc to be sure. Will try later today as I am in USA right now.
